server/src/main/scala/io/delta/sharing/server/config/ServerConfig.scala

```diff
@@ -76,7 +80,9 @@ case class ServerConfig(
       stalenessAcceptable = false,
       evaluatePredicateHints = false,
       evaluateJsonPredicateHints = false,
-      requestTimeoutSeconds = 30
+      requestTimeoutSeconds = 30,
+      queryTablePageSizeLimit = 500,
```

Comment:
Will this limit be applied to every request to the server?  Will pagination be enabled automatically for all requests? 
If the answer is yes, then is 10k still too small?

Reply:
Yes for both questions. The request timeout for OSS server is 30 seconds, and we don't do parallel signing. Not sure my projection is accurate here, if it takes 1ms to sign each file then I think the limit is better to be less than 25k (since we still need some time to load table and unpack delta logs)?

Reply:
10k sounds good, since this is the reference server.

I was thinking that these are the changes in DeltaSharingClient.